docs/Autotune.md

```diff
@@ -1,40 +1,41 @@
-# Autotune
+# Autotune (Rev. 2015-03-04)
 
 Autotune helps to automatically tune your multirotor.
 
 ## Configuration.
 
 Autotune only works in HORIZON or ANGLE mode, before using auto-tune it's best you setup so there is as little drift as possible.
-Autotuning is best on a full battery in good flying conditions, i.e. no or minimal wind. Autotune does not support 
-pid_controller 2 (pid_controller 0 is the Cleanflight default).
+Autotuning is best on a full battery in good flying conditions, i.e. no or minimal wind. Autotune does not support pid_controller 2  or higher (pid_controller 0 is the Cleanflight default, pid_controller 1 will work for autotune as well).
 
-Configure a two position switch on your transmitter to activate the AUTOTUNE and (HORIZON or ANGLE) modes using the auxiliary configuration.
-You may find a momentary switch more suitable than a toggle switch.
+Configure a two position switch on your transmitter to activate the AUTOTUNE mode. Autotune may be done in ether one of the both only, HORIZON or  ANGLE mode (will then apply on both modes). 
 
 
 ## Using autotuning
 
 Turn off the autotune switch.  If the autotune switch is on while not armed the warning LED will flash and you cannot arm.
 
-1. Launch the multirotor.
+1.	Launch the multirotor.
 
-1. Turn on/hold the autotune switch on your transmitter. Observe roll left/right.  A beep code will sound on the beeper.
+2.	Phase 1: ROLL/YAW autotune. 
```

Comment:
These points were all numbered using "1." in the original so that adding or removing entries doesn't require all the others to be renumbered (which creates diffs that don't need to be there). Markdown automatically adds the correct numbers in for display.